test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,174 @@
+             its1            its1-cluster    its1-admin      
+             kind-kubeflex   kind-kubeflex   kind-kubeflex   
+   *         wds1            wds1-cluster    wds1-admin      default
+             wds2            wds2-cluster    wds2-admin      default
+   ```
+
+   Second, create the binding Policy and namespace for your experiment using the following commands:
+
+   ```bash
+   cd $CL2_DIR/testing/load/
+   kubectl apply -f long-duration-exp-setup.yaml
+   ```
+
+   Lastly, in the following set ``--kubeconfig`` flag to the path of the kubeconfig file of your cluster and use the newly added Kubestellar provider (`--provider=ks`) and run the following command to create your workload:
+
+   ```bash
+   cd $CL2_DIR
+   go run cmd/clusterloader.go --testconfig=./testing/load/long-duration-exp-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
+   ```
+    
+   The generated workload is a pod that sleeps for 20 seconds([see here](workloads/long-duration-exp-workload.yaml)) and created using clusterloader `RSteppedLoad` tuningSet (burstSize=1 and stepDelay=60 sec).
+
+
+5. Run the metrics collection script: collect the creation and status_update timestamps for the benchmark workload objects proceed as following
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash
+   cd .$HOME/kubestellar/test/performance/common
+   ```
+
+   b) Run the metrics collection script:
+
+   ```bash 
+   python3 metrics_collector.py <kubeconfig> <wds-context-name> <its-context-name> <wec-context-name> <number-of-namespaces> <output-directory> <exp-type> <number-of-pods> <watch-interval>
+   ```
+
+   For example:
+   
+   ```bash 
+   python3 metrics_collector.py $HOME/.kube/config wds1 its1 cluster1 1 $HOME/data long_duration 10 30
+   ```
+
+   Observations: 
+      - The execution of the above python script requires the installation of the dependencies listed in `requirements.txt`.
+      - There is a custom controller that deletes a pod after reaching the completed state 
+      - The workload generation function created at `step-4` will launch another pod after 1 minute with a different name.
+
+   Below is a detailed explanation of the input parameters:
+   - `kubeconfig`: path to the kubeconfig file, e.g., `$HOME/.kube/config`
+   - `wds-context-name`: name of the context for the target WDS, e.g., `wds1`
+   - `its-context-name`: name of the context for the target ITS, e.g., `its1`
+   - `wec-context-name`: name of the context for the target WEC, e.g., `cluster1`
+   - `number-of-namespaces`: number of namespaces created in your experiment, e.g., `1`
+   - `output-directory`: path to the directory for the output data files, e.g., `$HOME/data`
+   - `exp-type`: experiment type (e.g., `short_duration` or `long_duration`)
+   - `number-of-pods`: total number of workload objects to be created in your experiment
```

Comment:
Is this counting pods or general objects?

Reply:
Fixed - it's counting pods.